src/test/java/seedu/address/model/book/UniqueBookListTest.java

```diff
@@ -166,4 +167,9 @@ public void asUnmodifiableObservableList_modifyList_throwsUnsupportedOperationEx
         assertThrows(UnsupportedOperationException.class, ()
             -> uniqueBookList.asUnmodifiableObservableList().remove(0));
     }
+
+    @Test
+    public void iterator_returnsIterator() {
```

Comment:
Feels redundant and method name sounds weird since it doesn't follow the featureUnderTest_testScenario_expectedBehavior() coding style

Reply:
Shall remove it, initially added to increase code coverage but I also feel that it is not a functional function to be tested. Maybe in the future, we can come up with a test to implicitly iterate through the `UniqueBookList` and therefore invoke this method implicitly.